### DISCUSSION TITLE
Accessing laragon from another local machine

### DISCUSSION THREAD

**Discussion by mjs1994** [leokhoa/laragon#476] [General]
Hey 👋 

What is the best method is to access my Laragon instance on my office PC from my laptop which is on the same network - I'm currently just using remote desktop and wondering if there is a more optimal method?

Cheers!

**Comment by veeroo18** [comment]
I am also looking for the same

**Reply by yonesmontja** [reply]
i am working in the same situation like yours. here is my case: i configure to run moodle platform in laragon environment and other pc can access the web server in the same local area network. and it's working pretty good. I simply put my pc's IP in the config.php file in moodle server.

**Reply by yonesmontja** [reply]
now i'm trying to run my own laravel web app and make other pc or machines to access my pc running laravel app in the same lan.

**Comment by loomy2021** [comment]
@veeroo18 it depends on your configuration, on the "laragon server" setup the firewall that it allows to communicate with network clients and then setup your nginx or apache to allow clients from within the same network.
That's it.

**Reply by hsimon78** [reply]
Hello,
By default, parameters deny access to all except 127.0.0.1.
If you use nginx you have to modify a couple of files (template) to change that, according to your security level requested. If you are in a private retwork you can specify "allow allé instead od "deny all".
Files to modify are :

- nginx.sites-enabled.00-default.SSL.manifest.tpl
- nginx.sites-enabled.00-default.manifest.tpl
- laragon/etc/nginx/sites-enabled/00-default.conf
- laragon\usr\tpl\nginx.sites-enabled.00-default.manifest.tpl 
Regards.

**Reply by supperball** [reply]
Thanks you, working for me
